theme/scripts/cart.ts

```diff
@@ -0,0 +1,110 @@
+import { handleFetchJSONResponse } from './utils';
+import activatable from './activatable';
+
+export default(function() {
+  const Cart = {
+    SELECTORS: {
+      addToCartForm: "[data-add-to-cart-form]",
+      addToCartButton: "[data-add-to-cart-button]",
+      navCartItemCount: "[data-nav-cart-item-count]",
+    },
```

Comment:
In this case,  what does `selector` mean?

Reply:
The way I am thinking of it is:

SELECTORS: Attributes used to select elements from the DOM
ATTRIBUTES: Attributes applied to the DOM (e.g. disabled attribute of data-active="true")

Reply:
Ahhhh ok thank you :)